concepticondata/conceptlists/Carvalho-2021-206.tsv

```diff
@@ -0,0 +1,207 @@
+Carvalho-2021-206-86	86	turtle	112	TURTLE
+Carvalho-2021-206-87	87	ant	587	ANT
+Carvalho-2021-206-88	88	isula (ant sp.)		
+Carvalho-2021-206-89	89	termite	883	TERMITE
+Carvalho-2021-206-90	90	wasp	1517	WASP
+Carvalho-2021-206-91	91	mosquito	1509	MOSQUITO
+Carvalho-2021-206-92	92	bee/honey		
+Carvalho-2021-206-93	93	fly	1504	FLY (INSECT)
+Carvalho-2021-206-94	94	butterfly	1791	BUTTERFLY
+Carvalho-2021-206-95	95	cockroach	1575	COCKROACH
+Carvalho-2021-206-96	96	spider	843	SPIDER
+Carvalho-2021-206-97	97	scorpion	1538	SCORPION
+Carvalho-2021-206-98	98	tick	1527	TICK
+Carvalho-2021-206-99	99	louse	1392	LOUSE
+Carvalho-2021-206-100	100	worm1	1219	WORM
+Carvalho-2021-206-101	101	worm2	1219	WORM
+Carvalho-2021-206-102	102	fish (generic)	227	FISH
+Carvalho-2021-206-103	103	piraña (fish sp.)	317	PIRANHA
+Carvalho-2021-206-104	104	crab	2348	CRAB
+Carvalho-2021-206-105	105	bat	1793	BAT
+Carvalho-2021-206-106	106	bird (sp.)1	937	BIRD
+Carvalho-2021-206-107	107	bird (sp.)2	937	BIRD
+Carvalho-2021-206-108	108	bird (sp.)3	937	BIRD
+Carvalho-2021-206-109	109	hummingbird	1589	HUMMINGBIRD
+Carvalho-2021-206-110	110	duck	1360	DUCK
+Carvalho-2021-206-111	111	eagle	1905	EAGLE
+Carvalho-2021-206-112	112	owl	735	OWL
+Carvalho-2021-206-113	113	tree	906	TREE
+Carvalho-2021-206-114	114	palm (sp.)	1181	PALM TREE
+Carvalho-2021-206-115	115	cane	659	REED
+Carvalho-2021-206-116	116	branch	1531	BRANCH
+Carvalho-2021-206-117	117	wood, stick	1295	STICK
+Carvalho-2021-206-118	118	seed	714	SEED
+Carvalho-2021-206-119	119	shell/husk		
+Carvalho-2021-206-120	120	flower	239	FLOWER
+Carvalho-2021-206-121	121	Leaf	628	LEAF
+Carvalho-2021-206-122	122	Coca	137	COCA
+Carvalho-2021-206-123	123	sweet potato	159	SWEET POTATO
+Carvalho-2021-206-124	124	bitter manioc		
+Carvalho-2021-206-125	125	manioc	927	MANIOC
+Carvalho-2021-206-126	126	maize	506	MAIZE
+Carvalho-2021-206-127	127	smoked (meat)		
+Carvalho-2021-206-128	128	pepper	1385	PEPPER
+Carvalho-2021-206-129	129	achiote	349	ACHIOTE
+Carvalho-2021-206-130	130	medicine	1372	MEDICINE
+Carvalho-2021-206-131	131	sun	1343	SUN
+Carvalho-2021-206-132	132	moon	1313	MOON
+Carvalho-2021-206-133	133	night	1233	NIGHT
+Carvalho-2021-206-134	134	yesterday	1174	YESTERDAY
+Carvalho-2021-206-135	135	wind	960	WIND
+Carvalho-2021-206-136	136	rainbow	1733	RAINBOW
+Carvalho-2021-206-137	137	lake	624	LAKE
+Carvalho-2021-206-138	138	water	948	WATER
+Carvalho-2021-206-139	139	rain	658	RAIN (PRECIPITATION)
+Carvalho-2021-206-140	140	fire/firewood	2675	FIRE OR FIREWOOD
+Carvalho-2021-206-141	141	fire/firewood2	2675	FIRE OR FIREWOOD
+Carvalho-2021-206-142	142	hole	1667	HOLE
+Carvalho-2021-206-143	143	stone	857	STONE
+Carvalho-2021-206-144	144	pain	1783	PAIN
+Carvalho-2021-206-145	145	to be/have	1579	BE
+Carvalho-2021-206-146	146	to arrive	1387	ARRIVE
+Carvalho-2021-206-147	147	to come	1446	COME
+Carvalho-2021-206-148	148	to return	581	COME BACK
+Carvalho-2021-206-149	149	to walk	1443	WALK
+Carvalho-2021-206-150	150	to pass	558	PASS BY
```

Comment:
Why not PASS?

Reply:
I assumed that it should refer to the physical motion as indicated by the surrounding concepts. While PASS is used to refer to time.